src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
```

Comment:
Use 'this' instead of 'calculator' to refer to the current instance within the subtract method. This ensures that the method remains functional even if the context of 'this' changes. Example: this.result -= number;